ci/do_ci.sh

```diff
@@ -197,7 +197,7 @@ case "$1" in
         exit 0
     ;;
     *)
-        echo "must be one of [build,test,clang_tidy,test_with_valgrind,coverage,asan,tsan,docker]"
+        echo "must be one of [build,test,clang_tidy,test_with_valgrind,coverage,asan,tsan,docker,check_format,fix_format]"
```

Comment:
Consider moving this into a separate PR? (It's good to have this, but it's not related to the PR topic?)

Reply:
Moving to #257